layouts/hypergol.json

```diff
@@ -0,0 +1,59 @@
+{
```

Comment:
All files related to Hypergol should probably come with another PR — along with a web page explaining what the layout is about. WDYT ?

Reply:
Oops, at first the PR was dedicated to Hypergol, I just needed “a few changes” to make it work. Forgot to remove hypergol once it became clear there was a *lot* of work needed